go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/stolostron/multicloud-operators-application
 
-go 1.19
+go 1.20
```

Comment:
Hi, can you also upgrade these k8s apis to their latest patch version, `v0.24.11`:
```
	k8s.io/api 
	k8s.io/apiextensions-apiserver
	k8s.io/apimachinery
	k8s.io/client-go
```
I may have missed mentioning these in the other repo.